src/components/Header/Header.js

```diff
@@ -15,71 +24,123 @@ const Header = ({data: {loading, error, myProfile: profile}}) => {
+                        <Icon name="history" />
+                        Event Log
+                      </Dropdown.Item>
+                    </Dropdown.Menu>
+                  </Dropdown>
+                )}
+                <Dropdown
+                  trigger={
+                    <>
+                      <Image avatar src={picUrl} alt={picAlt} />
+                      {profile.username}
+                    </>
+                  }
+                  className="link item"
+                >
                   <Dropdown.Menu>
-                    <Dropdown.Item as={Nav} to="/tokens">
-                      <Icon name="key" />
-                      Developer Tokens
-                    </Dropdown.Item>
-                    <Dropdown.Item as={Nav} to="/cavatica-projects">
-                      <Icon name="folder open" />
-                      Cavatica Projects
+                    <Dropdown.Item as={Nav} to="/profile">
+                      <Icon name="user" />
+                      Profile
                     </Dropdown.Item>
-                    <Dropdown.Item as={Nav} to="/events">
-                      <Icon name="history" />
-                      Event Log
+                    <Dropdown.Item
+                      as={Nav}
+                      to="/logout"
+                      onClick={() => {
+                        setLoggedIn(false);
+                        localStorage.removeItem('accessToken');
+                        localStorage.removeItem('idToken');
+                      }}
+                    >
+                      <Icon name="log out" />
+                      Logout
                     </Dropdown.Item>
                   </Dropdown.Menu>
                 </Dropdown>
-              )}
-              <Dropdown
-                trigger={
-                  <>
-                    <Image avatar src={picUrl} alt={picAlt} />
-                    {profile.username}
-                  </>
-                }
-                className="link item"
-              >
-                <Dropdown.Menu>
-                  <Dropdown.Item as={Nav} to="/profile">
-                    <Icon name="user" />
-                    Profile
-                  </Dropdown.Item>
-                  <Dropdown.Item
-                    as={Nav}
-                    to="/logout"
-                    onClick={() => {
-                      setLoggedIn(false);
-                      localStorage.removeItem('accessToken');
-                      localStorage.removeItem('idToken');
-                    }}
-                  >
-                    <Icon name="log out" />
-                    Logout
-                  </Dropdown.Item>
-                </Dropdown.Menu>
-              </Dropdown>
-            </Menu.Menu>
-          </>
-        )}
-      </Container>
-    </Menu>
+              </Menu.Menu>
+            </>
+          )}
+        </Container>
+      </Menu>
+      <Segment vertical basic className="footer">
```

Comment:
the footer needs moved to its own component and not be part of the header component